spec/models/waste_carriers_engine/registration_spec.rb

```diff
@@ -520,9 +530,9 @@ module WasteCarriersEngine
               expect(registration.metaData).to allow_event :renew
             end
 
-            it "expires when it reaches the expiry date in the UK" do
-              # Skip ahead to the start of the day a reg should expire
-              Timecop.freeze(Time.find_zone("London").local(2018, 10, 27, 0, 1))
+            it "expires when it reaches the expiry date plus 'grace window' in the UK" do
+              # Skip ahead to the start of the day a reg should expire, plus the
```

Comment:
Think this comment is missing an ending!

Reply:
Also, it expires when it hits the expiry date – it can still be renewed for another few days, but that doesn't mean it's not expired. So I'd prefer "it cannot be renewed" to "it expires". Same goes for the test on line 496.